src/components/ThemeProvider/__stories__/ThemeProvider.stories.js

```diff
@@ -72,3 +76,36 @@ export const CustomClassSelector = {
   render: ThemeProviderCustomClassTemplate.bind({}),
   name: "Custom class selector"
 };
+
+export const WithSystemTheme = {
+  render: () => {
+    const [shouldRenderThemeProvider, setShouldRenderThemeProvider] = useState(false);
+
+    return (
+      <Flex direction={Flex.directions.ROW} gap={Flex.gaps.LARGE}>
+        <Toggle
+          isSelected={shouldRenderThemeProvider}
+          onChange={setShouldRenderThemeProvider}
+          name={"Render ThemeProvider"}
+        />
+        {shouldRenderThemeProvider && (
+          <ThemeProvider
+            theme={{
+              name: "with-system-theme",
+              colors: {
+                [ThemeProvider.systemThemes.LIGHT]: {
+                  [ThemeProvider.colors.primaryColor]: "green",
+                  [ThemeProvider.colors.primaryHoverColor]: "darkgreen"
+                }
+              }
```

Comment:
since you're showcasing the `systemTheme`, you can remove the `colors`, it will get it from the default dark-light styles

Reply:
I think it adds to better visualization